src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/NoOpRestoreUtilities.cs

```diff
@@ -111,7 +111,7 @@ internal static string GetCacheFilePath(RestoreRequest request, LockFile lockFil
         /// This method verifies that the props/targets files and all the packages written out in the lock file are present on disk
         /// This does not account if the files were manually modified since the last restore
         /// </summary>
-        internal static bool VerifyAssetsAndMSBuildFilesAndPackagesArePresent(RestoreRequest request)
+        internal static bool VerifyAssetsAndMSBuildFilesAndLockFilesAndPackagesArePresent(RestoreRequest request)
```

Comment:
This method name has gotten kind of crazy. 

How about `VerifyRestoreOutput` and updating the comment to clearly indicate what is being validated.

Reply:
Thanks! Applied.